Lab 8 - Arrays.md

```diff
@@ -275,3 +275,32 @@ Sotib olingan soyabonlar sonini chop eting.
 | Input | Output |
 | - | - |
 | `2 5 1`<br>`2 0`<br>`1 1`<br>`2 0`<br>`1 1`<br>`2 1` | `2` |
+
+### Masala 12.
+
+### Array birlashtirish
+
+###### Description
+
+>  Sizga ikkita saralangan bog'langan ro'yxatlar array 1 va array 2 beriladi.
+>  Ikki arrayni bitta tartiblangan arrayda birlashtiring. Array birinchi ikkita array elementrini birlashtirish orqali tuzilishi kerak.
+>  Birlashtirilgan bog'langan arrayning boshini qaytaring.  

```

Comment:
- bog'langan so'zi nima manoni anglatadi?
- Masala sharti sal tushinarsizroq ekan. Shuni to'g'irlab qo'ysangiz.